app/views/coronavirus_form/know_nhs_number.html.erb

```diff
@@ -4,6 +4,8 @@
 
 <% content_for :meta_tags do %>
   <meta name="description" content="<%= t("coronavirus_form.questions.know_nhs_number.title") %>" />
+  # stop iOS treating the NHS number as a phone number
```

Comment:
These comments are being rendered to the user.  This will fix that.
```suggestion
  <%# # stop iOS treating the NHS number as a phone number %>
```

Reply:
Was too hasty with this one. Need to take a breather, thanks.